pandas/io/tests/test_sql.py

```diff
@@ -682,6 +683,18 @@ def test_date_and_index(self):
         self.assertTrue(issubclass(df.IntDateCol.dtype.type, np.datetime64),
                         "IntDateCol loaded with incorrect type")
 
+    def test_sqlite_time(self):
+        # test support for datetime.time
+        df = DataFrame([time(9, 0, 0), time(9, 1, 30)], columns=["a"])
```

Comment:
The `if flavor == 'sqlite'` is to prevent that it is tested on mysql (as the PR implements it only for sqlite, not for mysql, which is deprecated anyway)